app/controllers/courses_controller.rb

```diff
@@ -33,6 +33,7 @@ def create
              status: 404
       return
     end
+    # TODO: Update the below line
```

Comment:
We need to add a conditional to make sure that if a rollback occured, we should return json with the error messages. We might have to update the create method of `CourseCreationManager`.